core/tauri-build/src/lib.rs

```diff
@@ -514,54 +490,21 @@ pub fn try_build(attributes: Attributes) -> Result<()> {
   let out_dir = PathBuf::from(std::env::var("OUT_DIR").unwrap());
 
   manifest::check(&config, &mut manifest)?;
-  let mut plugin_manifests = acl::get_plugin_manifests()?;
-  for (name, plugin) in attributes.inlined_plugins {
-    let plugin_out_dir = out_dir.join("plugins").join(name);
-
-    let mut permission_files = if plugin.commands.is_empty() {
-      Vec::new()
-    } else {
-      tauri_utils::acl::build::autogenerate_command_permissions(
-        &plugin_out_dir,
-        plugin.commands,
-        "",
-      );
-      tauri_utils::acl::build::define_permissions(
-        &plugin_out_dir.join("*").to_string_lossy(),
-        name,
-        &plugin_out_dir,
-      )?
-    };
-
-    if let Some(pattern) = plugin.permissions_path_pattern {
-      permission_files.extend(tauri_utils::acl::build::define_permissions(
-        pattern,
-        name,
-        &plugin_out_dir,
-      )?);
-    } else {
-      let default_permissions_path = Path::new("permissions").join(name);
-      println!(
-        "cargo:rerun-if-changed={}",
-        default_permissions_path.display()
-      );
-      permission_files.extend(tauri_utils::acl::build::define_permissions(
-        &default_permissions_path
-          .join("**")
-          .join("*")
-          .to_string_lossy(),
-        name,
-        &plugin_out_dir,
-      )?);
-    }
 
-    let manifest = tauri_utils::acl::plugin::Manifest::new(permission_files, None);
-    plugin_manifests.insert(name.into(), manifest);
-  }
+  let mut acl_manifests = acl::get_manifests_from_plugins()?;
+  acl_manifests.insert(
+    "".into(),
```

Comment:
this seems sort of magical key and can be easily forgot about? we should add an explicit field or marker for it

Reply:
you're right, pushed